### DISCUSSION TITLE
Is there any version of nemo-guardrails for cpu only

### DISCUSSION THREAD

**Discussion by hieudx149** [NVIDIA-NeMo/Guardrails#365] [General]
when i run pip install nemoguardrails, it auto install all libs for GPU setup

**Comment by drazvan** [comment]
Hi @hieudx149! As of `0.7.1` when we made the switch to `fastembed`, it should no longer install all the GPU-related dependencies.

**Reply by hieudx149** [reply]
so how can i load a sentence-transformer models ?
